core/templates/components/rubrics-editor/rubrics-editor.directive.ts

```diff
@@ -73,7 +73,7 @@ angular.module('oppia').directive('rubricsEditor', [
           };
 
           ctrl.saveExplanation = function(difficulty, index) {
-            if (difficulty === 'Medium' && index === 0) {
+            if (difficulty === SKILL_DIFFICULTIES[1] && index === 0) {
```

Comment:
The naming / data structure should permit calling this SKILL_DIFFICULTY_MEDIUM or SKILL_DIFFICULTY.MEDIUM -- don't use stuff like SKILL_DIFFICULTIES[1] because no one will know what the significance of 1 is.

If you need the skill difficulties in order for something else then maybe you'll need to define that constant as SKILL_DIFFICULTIES = [SKILL_DIFFICULTY_EASY, SKILL_DIFFICULTY_MEDIUM, SKILL_DIFFICULTY_HARD] instead.

Reply:
Done.